query/src/interpreters/interpreter_user_stage_create.rs

```diff
@@ -55,7 +55,14 @@ impl Interpreter for CreateUserStageInterpreter {
         if user_stage.stage_type == StageType::Internal {
             let prefix = format!("stage/{}/", user_stage.stage_name);
             let op = self.ctx.get_storage_operator()?;
-            op.object(&prefix).create().await?;
+            op.object(&prefix).create().await.or_else(|e| {
+                if e.kind() == std::io::ErrorKind::AlreadyExists {
+                    // if target path already exist, just ignore the error
```

Comment:
It's a bug of opendal that should be addressed.  It's OK to workaround it here and I will fix it ASAP. Can you help leave a TODO(xuanwo) here?

Reply:
Fixed in https://github.com/datafuselabs/opendal/pull/234/files

Reply:
PTAL https://github.com/datafuselabs/databend/pull/4747

Reply:
Let's rename this workaround :heart:

Reply:
got it, thanks!